packages/frontend/core/src/components/affine/auth/user-plan-button.tsx

```diff
@@ -0,0 +1,34 @@
+import { SubscriptionPlan } from '@affine/graphql';
+import Tooltip from '@toeverything/components/tooltip';
+import { useSetAtom } from 'jotai';
+import { useCallback } from 'react';
+
+import { openSettingModalAtom } from '../../../atoms';
+import { useUserSubscription } from '../../../hooks/use-subscription';
+import * as styles from './style.css';
+
+export const UserPlanButton = () => {
+  const [subscription] = useUserSubscription();
+  const plan = subscription?.plan ?? SubscriptionPlan.Free;
+
+  const setSettingModalAtom = useSetAtom(openSettingModalAtom);
+  const handleClick = useCallback(
+    (e: React.MouseEvent<HTMLDivElement>) => {
+      e.stopPropagation();
```

Comment:
This line of code is to prevent clicking the button from jumping to the account settings.

Clicking this button in the workspace list will open the settings page and close the workspace list. Should this be expected behavior?